packages/sonarwhal/src/lib/cli/init.ts

```diff
@@ -19,10 +20,52 @@ import { debug as d } from '../utils/debug';
 import * as logger from '../utils/logging';
 import * as resourceLoader from '../utils/resource-loader';
 import { generateBrowserslistConfig } from './browserslist';
+import { NpmPackage } from '../types';
 
 const debug: debug.IDebugger = d(__filename);
 const defaultFormatter = 'summary';
 
+const packageExists = () => {
+    const packagePath = path.join(process.cwd(), 'package.json');
+
+    return fs.existsSync(packagePath); // eslint-disable-line no-sync
+};
+
+const installRules = (rules) => {
+    const global: boolean = !packageExists();
+
+    const packages: Array<string> = [];
+
+    for (const [key, value] of Object.entries(rules)) {
+        if (value !== 'off') {
+            packages.push(`@sonarwhal/rule-${key}`);
+        }
+    }
+
+    const command: string = `npm install ${packages.join(' ')}${global ? ' -g' : ''}`;
+
+    try {
+        logger.log(`Running command ${command}`);
+
+        const result: SpawnSyncReturns<Buffer> = spawnSync(command, { shell: true });
+
+        if (result.status !== 0) {
+            throw new Error(result.output[2].toString());
+        }
+
+        logger.log('Packages intalled successfully');
+    } catch (err) {
+        /*
+         * There was an error installing packages.
+         * Show message to install packages manually.
+         */
+        logger.error(err);
+        logger.log(`Something when wrong installing package, please run:
+${process.platform === 'linux' ? 'sudo ' : ''}${command}
```

Comment:
What's the value for Mac platforms? Maybe this should be `process.platform !== 'windows'`